### PR TITLE
Connected toolbar to nav & edited header

### PR DESCRIPTION
**Pull request by R303Vermeulen**
#20 
Connected toolbar to nav, also creating dummy screens for profile, forums, search, and settings.
Moved toolbar icons folder to assets instead of comps/toolbar
Updated Header to have the same stylings as screen
Created HeaderBackButton Component to replace the native backbutton

### CODE REVIEW

File: Eat_and_Greet/App.tsx [hunk 1]
```diff
@@ -1,20 +1,37 @@
...
 
 const Stack = createNativeStackNavigator();
 
 export default function App() {
   return (
     <NavigationContainer>
-      <Stack.Navigator>
+      <Stack.Navigator
+        screenOptions={{
+          ...navHeaderStyles,
```
Comment (MorrowCameron) [inline Eat_and_Greet/App.tsx:23]: What is the ...? Is that just git or is there a reason?

File: Eat_and_Greet/App.tsx [hunk 2]
```diff
@@ -1,20 +1,37 @@
...
+          headerBackTitleVisible: true,
+          headerLeft: () => <HeaderBackButton />,
+        }}
+      >
         <Stack.Screen name="Login" component={LoginScreen} />
```
Comment (MorrowCameron) [inline Eat_and_Greet/App.tsx:28]: I like this, worth considering doing this in a for loop with an array of tuples or something. If not done now, can make a ticket to do later.

File: Eat_and_Greet/src/comps/HeaderBackButton/index.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    alignItems: "center",
+  },
+  text: {
+    fontFamily: "Courier New",
+    fontWeight: "bold",
+    color: "black",
+    fontSize: 24,
+    paddingTop: 2
+  },
+  image: {
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/comps/HeaderBackButton/index.styles.tsx:15]: nit: image -> imageSizing

File: Eat_and_Greet/src/comps/HeaderBackButton/index.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { TouchableOpacity, Text, Image } from 'react-native';
+import localStyles from './index.styles';
+import { useNavigation, useRoute } from '@react-navigation/native';
+
+const HeaderBackButton = () => {
+
+    const navigation = useNavigation();
+
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/comps/HeaderBackButton/index.tsx:9]: Consider deleting this whitespace line

File: Eat_and_Greet/src/style.tsx [hunk 5]
```diff
@@ -1,25 +1,37 @@
...
+    backgroundColor: "#7286D3",
+    alignItems: "center",
+    justifyContent: "flex-start",
+  },
+});
 
-export default style;
\ No newline at end of file
+export const navHeaderStyles = StyleSheet.create({
+  headerStyle: {
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/style.tsx:26]: nit: headerStyle -> headerColor

Comment (R303Vermeulen) [reply Eat_and_Greet/src/style.tsx:26]: headerStyle is predetermined naming with native navigation

File: Eat_and_Greet/App.tsx (hunk 2 above)
Comment (R303Vermeulen) [reply Eat_and_Greet/App.tsx:28]: used a map function & tuple array to change

File: Eat_and_Greet/src/comps/Toolbar/toolbar.tsx [hunk 6]
```diff
@@ -1,16 +1,44 @@
...
 import styles from './style';
+import IconButton from './iconButton';
+
 export function Toolbar() {
+
+  const navigation = useNavigation();
+
+  const route = useRoute();
+
+  const homeImg = route.name === 'Home'
```
[the remark this answers is not in the document]
Comment (R303Vermeulen) [reply Eat_and_Greet/src/comps/Toolbar/toolbar.tsx:13]: Ran into an issue so putting in separate ticket

File: Eat_and_Greet/App.tsx [hunk 7]
```diff
@@ -1,20 +1,41 @@
...
-        <Stack.Screen name="Home" component={HomeScreen} />
-        <Stack.Screen name="Onboarding" component={OnboardingScreen} />
+      <Stack.Navigator
+        screenOptions={{
+          ...navHeaderStyles,
+          headerBackTitleVisible: true,
+          headerLeft: () => <HeaderBackButton />,
+        }}
+      >
+        {stackArray.map((stackObj) => (
```
Comment (MorrowCameron) [inline Eat_and_Greet/App.tsx:36]: I love this and I love you.

Comment (MorrowCameron) [reply Eat_and_Greet/App.tsx:36]: It looks so much cleaner!

File: Eat_and_Greet/src/comps/HeaderBackButton/index.styles.tsx [hunk 8]
```diff
@@ -0,0 +1,21 @@
+import { StyleSheet } from "react-native";
+
+const localStyles = StyleSheet.create({
+  container: {
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/comps/HeaderBackButton/index.styles.tsx:4]: Don't adore the naming here, but I'm not gonna block the release. In the future avoid just container as I'd like to know what's being contained or why it's special.

File: Eat_and_Greet/src/comps/Toolbar/toolbar.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (MorrowCameron) [reply Eat_and_Greet/src/comps/Toolbar/toolbar.tsx:13]: Love it.

File: Eat_and_Greet/src/screens/ForumsScreen/index.tsx [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+import { StatusBar } from 'expo-status-bar';
+
+import { Toolbar } from '../../comps/Toolbar/toolbar';
+
+import styles from '../../style';
+
+function ForumsScreen() {
+    return (
+      <View style={styles.ScreenContainer}>
+        <View style={styles.ContentContainer}>
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/screens/ForumsScreen/index.tsx:11]: Love the split descriptions of the containers!

File: Eat_and_Greet/src/screens/HomeScreen/index.tsx [hunk 10]
```diff
@@ -10,11 +10,13 @@ import styles from '../../style';
 function HomeScreen() {
     return (
       <View style={styles.ScreenContainer}>
+        <View style={styles.ContentContainer}>
+          <StatusBar style="auto" />
+          <TextBox placeholder="Type ..." />
+          <Text>Home Screen Bruh</Text>
+          <Text>Welcome to our app!</Text>
+        </View>
         <Toolbar />
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/screens/HomeScreen/index.tsx:19]: <3

File: Eat_and_Greet/src/style.tsx (hunk 5 above)
Comment (MorrowCameron) [reply Eat_and_Greet/src/style.tsx:32]: Sounds good!

**Review by MorrowCameron — COMMENTED** [review]
Overall Great Job! Keep it up!

File: Eat_and_Greet/App.tsx (hunk 7 above)
Comment (R303Vermeulen) [reply Eat_and_Greet/App.tsx:36]: <3 happy with it

**Pull request merged**